better_figures_and_images/better_figures_and_images.py

```diff
@@ -45,6 +45,9 @@ def content_object_init(instance):
                     img_path = img_path[10:]
                 elif img_path.startswith('/static'):
                     img_path = img_path[7:]
+                elif img_path.startswith('data:image'):
+                    # Image is encoded in-line (not a file).
+                    break
```

Comment:
err... Shouldn't this be `continue`? `break` will stop processing of other images.